gulpfile.js

```diff
@@ -1,42 +1,31 @@
-const os = require('os');
-const gulp = require('gulp');
-const sourcemaps = require('gulp-sourcemaps');
-const source = require('vinyl-source-stream');
-const buffer = require('vinyl-buffer');
-const browserify = require('browserify');
-const babel = require('babelify');
-const nodemon = require('gulp-nodemon');
-const prettify = require('gulp-jsbeautifier');
+const gulp = require('gulp')
+const sourcemaps = require('gulp-sourcemaps')
+const source = require('vinyl-source-stream')
+const buffer = require('vinyl-buffer')
+const browserify = require('browserify')
+const babel = require('babelify')
+const nodemon = require('gulp-nodemon')
+const eslint = require('gulp-eslint')
 
 // Gulp Tasks
-gulp.task('beautify', beautify);
-gulp.task('build', build);
-gulp.task('watch', ['build'], watch);
+gulp.task('lint', lint)
+gulp.task('build', ['lint'], build)
+gulp.task('watch', ['build'], watch)
 
-gulp.task('default', ['watch']);
+gulp.task('default', ['watch'])
 
 const globsToBuild = [
   'public/scripts/*.js'
-];
+]
 
-const globsToBeautify = [
-  '*.js',
-  'public/**/*.js',
-  'public/**/*.html',
-  'public/**/*.css',
-  '!public/bundle.js*' // don't beautify our bundled scripts
-];
-
-// Task implementations
-function beautify() {
-  gulp.src(globsToBeautify, {
-      base: './'
-    })
-    .pipe(prettify({
-      eol: os.EOL, // \r\n on Windows, \n on POSIX
+function lint() {
+  return gulp.src(['*.js', 'public/scripts/*.js'])
```

Comment:
do you need to return this or not? I had an issue where returning from `build` caused me some issues, but I don't really understand gulp that well